grants_tagger/models/create_model_xlinear.py

```diff
@@ -0,0 +1,31 @@
+import ast
+from grants_tagger.utils import load_pickle
+from grants_tagger.models.mesh_xlinear import MeshXLinear
+
+
+class ApproachNotImplemented(Exception):
+    pass
+
+
+def create_model(approach, parameters=None):
```

Comment:
I know it's pretty obvious what this function does, but for completeness sake a docstring should be added here.